account_move_tier_validation/models/account_move.py

```diff
@@ -28,7 +28,15 @@ def _get_validation_exceptions(self, extra_domain=None, add_base_exceptions=True
         # otherwise editing manually the values on lines dirties the field at onchange
         # since it's not in readonly because readonly="not(review_ids)", it's then
         # sent at save, and will override the values set by the user
-        return res + ["amount_total"]
+        # The other exclusions are needed to be able to generate the pdf
+        # and send the invoice by email
+        am_exceptions = [
+            "needed_terms_dirty",
+            "is_manually_modified",
+            "is_move_sent",
+            "sending_data",
+        ]
```

Comment:
Should we add `["amount_total"]` in am_exceptions? and return with res + am_exceptions
```suggestion
            "amount_total",
        ]
        return res + am_exceptions
```

Reply:
Yes, that would be more clean fix